php/primes.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+function isPrime($num, $primes) {
+    foreach($primes as $i) {
+        if($num % $i == 0)
+            return false;
+    }
+
+    return true;
+}
+
+$primes = array(2, 3);
+
+for($i = 4; $i <= 100000; $i = $i + 1)
+    if(isPrime($i, $primes))
+        array_push($primes, $i);
+
+?>
```

Comment:
:D , I have basic knowledge and that's it, but I can this language with me if you want.